apps/src/templates/lessonOverview/activities/Activity.jsx

```diff
@@ -23,7 +23,7 @@ export default class Activity extends Component {
         <h2 style={styles.activityHeader}>
           {i18n.activityHeader({
             activityName: activity.displayName,
-            activityTime: activity.time
+            activityDuration: activity.duration
```

Comment:
note that changing the names of the interpolation values here means that this call will likely break for translated strings until they get updated

Reply:
Thanks @Hamms . that's fine with us, because this feature hasn't launched yet.